tasks/update.yml

```diff
@@ -1,13 +1,22 @@
 ---
-- name: Run database updates.
-  command: "{{ drush_path }} updatedb -y"
+
+- name: Detect Drupal version.
+  command: drush status drupal-version --format=list
   args:
     chdir: "{{ drupal_core_path }}"
+  register: drupal_core_version_output
+
+- name: Set Drupal version.
+  set_fact:
+    drupal_core_version: "{{ drupal_core_version_output.stdout }}"
 
-# TODO: Import configuration if configured?
-# TODO: Other commands if configured?
+- name: Run post-update tasks.
+  command: "{{ item.command }}"
+  args:
+    chdir: "{{ item.chdir | default(omit) }}"
```

Comment:
I've encountered use-cases for the path not being `drupal_core_path`, and specifying it as a command option vs `chdir` seems partly a style preference to me.  Allowing override here also prevents accidentally excluding some command that doesn't have a path option.  I can't think of one off the top of my head, but that doesn't mean there isn't one...

I can also see the convenience of defaulting to `drupal_core_path`.

What about allowing override, but with a default to `drupal_core_path`?